### PR TITLE
Feature: Create Profile Page

### PR DESCRIPTION
**Pull request by jayciemcbrayer**
# [Create Profile Page](https://trello.com/c/yvz4VYqu/6-create-profile-page)

## What did I change?
As a user, I would like to see a page, so I can view my profile and progress.

## Test Result:
Screen Shot 2021-07-23 at 11 39 33 AM

## Lint Results:
Screen Shot 2021-07-23 at 11 41 16 AM

### Checklist
- [x] I acknowledge that all my contributions will be made under the project's license
- [x] I have made a material change to the repo (functionality, testing, spelling, grammar)
- [x] I have read the [Contribution Guidelines](https://github.com/aaronshivers/snapit-hacktoberfest/blob/main/CONTRIBUTING.md) and my PR follows them
- [x] I have titled the PR appropriately
- [x] I have updated my branch with the main branch
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added the necessary documentation about the functionality in the appropriate .md file
- [x] I have added inline documentation to the code I modified

If you have questions, please create a GitHub Issue in this repository.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -5,3 +5,4 @@ dist-ssr
 *.local
 .idea
 .eslintcache
+.eslintcache
```
Comment (aaronshivers) [inline .gitignore:8]: 🔧 It looks like `eslintcache` is already in the file.

File: README.md [hunk 2]
```diff
@@ -1,5 +1,16 @@
...
+
+Install Node Version:
+`nvm install v14.17.3`
+Install Yarn
+`https://classic.yarnpkg.com/en/docs/install/`
+Install Packages
+`yarn install`
+Run Application
+`yarn dev`
+
```
Comment (aaronshivers) [inline README.md:13]: ⛏️ Add an empty line between each instruction. Currently they're displaying in one line.

File: src/ProfilePage.css [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  left: 300px;
+  top: 85px;
+}
+
+.progressBar {
+  width: 30%;
+  position: relative;
+  left: 300px;
+  top: 90px;
+}
```
Comment (aaronshivers) [inline src/ProfilePage.css:24]: 🔧 Move this inside the `Profile` directory.

Comment (aaronshivers) [reply src/ProfilePage.css:24]: 🔧 Rename to `Profile.css`

**Comment by dylhack** [comment]
I'll review after @aaronshivers

File: src/pages/Profile/Profile.css [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+  left: 300px;
+  top: 85px;
+}
+
+.progressBar {
+  width: 30%;
+  position: relative;
+  left: 300px;
+  top: 90px;
+}
```
Comment (aaronshivers) [inline src/pages/Profile/Profile.css:24]: 🔧 Let's use Material-UI built-in layout tools instead of custom css.
https://material-ui.com/components/box/

**Review by aaronshivers — CHANGES_REQUESTED** [review]

File: src/pages/Profile/Profile.css (hunk 4 above)
Comment (jayciemcbrayer) [reply src/pages/Profile/Profile.css:24]: I would LOVE to do that, could you show me how, please?

**Comment by ssimmons05** [comment]
Closing this to open a new version of this PR.

**Pull request closed without merging**